sku.0/sys.server/compiled/game/script/gm/cmd.java

```diff
@@ -4439,6 +4439,38 @@ else if (command.equalsIgnoreCase("setWeather")) {
             return SCRIPT_CONTINUE;
         }
 
+        else if (command.equalsIgnoreCase("setCount")) {
+            
+            int count;
+            if(st.hasMoreTokens()) {
+                count = utils.stringToInt(st.nextToken());
+            } else {
+                sendSystemMessageTestingOnly(self, "Syntax: /admin setCount <integer> ");
+                return SCRIPT_CONTINUE;
+            }
+            if (count > 500) {
+                count = 500;
+            }
+            setCount(target, count);
+            sendConsoleMessage(self, "You have added " + count + " to " + target + " (" + getName(target) + ")");

```

Comment:
Not sure why these are console messages when standard pattern is system messages for admin command results. Not a big deal but curious deviation.